### ISSUE TITLE
Feature Request: Option to retain position in queue on editing

### ISSUE THREAD

**Issue by thunder175** [linuxgurugamer/KCT#77]
One small thing thats always kind of annoyed me is that if you edit a craft already in production, when you save the edits KCT automatically moves it to the bottom of the build queue. In the mid stages of the game when you have a lot of craft awaiting production, this is annoying and time consuming to put it back where you would like it in the prioritization list. 

A feature or radio button that would allow the crafts position to be retained in the build queue would be great! I say an option in the settings or a radio button on the craft edit menu would be better as it gives options to players who might prefer the default (current) behavior.

**Comment by vksavochkin** [comment]
Yeah, I too would like edited crafts to retain their position or go straight to the top (not bottom) of a queue if no position saved (frankly, even if there is).

P.S. Years playing with KCT and only now I noticed button "Move to Top" in queue craft options... facepalm.jpeg.

**Comment by thunder175** [comment]
I've been playing RO/RP1 and their highly integrated version of KCT has that feature. Not sure if it would be possible to share some of their code and integrate into the regular KCT baseline?
